### PR TITLE
Add inheritance test

### PR DESCRIPTION
**Pull request by franmomu**
Closes https://github.com/sonata-project/SonataDoctrineORMAdminBundle/issues/1370

It is based on https://github.com/sonata-project/SonataDoctrineORMAdminBundle/issues/1361

### CODE REVIEW

**Comment by franmomu** [comment]
🤔 @Devristo I tried to use your example from https://github.com/Devristo/sonata-admin-entity-inheritance, can you please look to see if there is something different? Maybe it's because posgresql.

**Comment by jordisala1991** [comment]
Why dont keep using sqlite?

**Comment by franmomu** [comment]
> Why dont keep using sqlite?

I just wanted to try if the fail was using posgresql (not as a permanent change), it would be interesting to add tests with different platforms to catch these things.

**Comment by Devristo** [comment]
> thinking @Devristo I tried to use your example from https://github.com/Devristo/sonata-admin-entity-inheritance, can you please look to see if there is something different? Maybe it's because posgresql.
> 
> **Update**: Looks like it fails only using posgresql.

Postgresql is indeed more strict about non-aggregated fields in group by than some others. I think more recent MySQL versions are as well: https://dev.mysql.com/doc/refman/5.7/en/group-by-handling.html .

I have no clue about sqlite though.

**Comment by franmomu** [comment]
Needs https://github.com/sonata-project/SonataDoctrineORMAdminBundle/pull/1399

File: tests/App/Admin/SubAdmin.php [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\DoctrineORMAdminBundle\Tests\App\Admin;
+
+use Sonata\AdminBundle\Admin\AbstractAdmin;
+use Sonata\AdminBundle\Datagrid\ListMapper;
+
+class SubAdmin extends AbstractAdmin
```
Comment (jordisala1991) [inline tests/App/Admin/SubAdmin.php:19]: missing extends generic type

**Review by VincentLanglet — APPROVED** [review]
Does this fail without the fix for #1361 ?

**Review by phansys — APPROVED** [review]

**Comment by franmomu** [comment]
> Does this fail without the fix for #1361 ?

It used to fail with the previous configuration I had for `PostgreSQL`, I guess it should also fail now.

**Review by jordisala1991 — APPROVED** [review]

**Comment by jordisala1991** [comment]
Thank you @franmomu

**Pull request merged**